magento/pwa-studio
# [doc] architecture overview

## Description

Provides an architectural overview of PWA Studio and its storefronts.

## Related Issue

Closes #1348 

## Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Run linting on the affected files: `npm run lint src/technologies/storefront-architecture/`
3. Verify no linting errors.
4. Build the preview site: `npm run develop`
5. Verify pages are generated and available in the TOC:
    * `technologies/storefront-architecture`
    * `technologies/storefront-architecture/buildtime`
    * `technologies/storefront-architecture/runtime`
